resumebuilder/src/components/Volunteer.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const Volunteer = props => {
+	const { applicant } = props;
+	const volunteer = applicant.volunteer.map(event => {
+		return (
+			<>
+				<p>{event.organization}</p>
+				<p>{event.position}</p>
+				<p>{event.website}</p>
+				<p>{event.startDate}</p>
+				<p>{event.endDate}</p>
+				<p>{event.summary}</p>
+				<p>{event.highlights}</p>
```

Comment:
You could try to map over the keys in your object to make your code a bit more DRY. Something like:

``` Object.keys(event).map(eventKey => {
     return <p>{event[eventKey]}</p>
}```

(not tested)

Reply:
Thanks, good call! I figured there was a better way! Next time ill catch that, annoying that didn't come to me immediately while i was writing that. I think that will work, if not slightly different syntax